spec/features/new_post_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature "New Post" do
+  scenario "User able to add a new post to Chitter" do
+    visit("/peeps/new")
+
+    fill_in("peep", with: "Hi Chitter!")
+    click_button("Post")
+
+    expect(page).to have_content "Hi Chitter!"
```

Comment:
Are there some more expectations you could include in this feature test, like expecting the page to display the time the peep was made?

Reply:
Or for the peep db to contain this peep?